misk-crypto/src/main/kotlin/misk/crypto/TestKeysets.kt

```diff
@@ -0,0 +1,194 @@
+package misk.crypto
+
+import com.google.crypto.tink.CleartextKeysetHandle
+import com.google.crypto.tink.JsonKeysetReader
+import com.google.crypto.tink.JsonKeysetWriter
+import com.google.crypto.tink.aead.KmsEnvelopeAead
+import misk.config.MiskConfig
+import java.io.ByteArrayOutputStream
+
+/**
+ * These test keys are used by the [FakeExternalKeyManager] class.
+ * *DO NOT USE IN ANY OTHER SITUATION*
+ */
+internal class TestKeysets {
+  companion object {
+    fun encryptSecret(key: MiskConfig.RealSecret<String>): MiskConfig.RealSecret<String> {
+      val kek = KmsEnvelopeAead(KeyReader.KEK_TEMPLATE, FakeMasterEncryptionKey())
+      val keyOutput = ByteArrayOutputStream()
+      val plaintextKey = CleartextKeysetHandle.read(JsonKeysetReader.withString(key.value))
+      plaintextKey.write(JsonKeysetWriter.withOutputStream(keyOutput), kek)
+      return MiskConfig.RealSecret(keyOutput.toString())
+    }
+    /**
+     * Created with tikney:
+     * `tinkey create-keyset --key-template AES256_GCM`
+     */
+    val AEAD = MiskConfig.RealSecret("""{
+    "primaryKeyId": 287541552,
```

Comment:
nit: Would be nice to have these all as blobs in a yaml so they are easiest to swap.